src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -1764,6 +1778,7 @@ private static RegexOptions OptionFromCode(char ch)
                 'd' => RegexOptions.Debug,
 #endif
                 'e' => RegexOptions.ECMAScript,
+                'a' => RegexOptions.AnyNewLine,
```

Comment:
Just noticed that the original proposal didn't suggest we add an inline option for AnyNewLine - it's reasonable to do but we'd need tests for it. It looks like we don't have a great set of tests for existing inline regex options other than just setting them once.